modules/sequence/src/main/scala/lucuma/odb/sequence/util/HashBytes.scala

```diff
@@ -71,9 +72,18 @@ object HashBytes {
       )
   }
 
+  given HashBytes[Timestamp] with {
+    def hashBytes(a: Timestamp): Array[Byte] =
+      HashBytes[Long].hashBytes(a.toEpochMilli)
+  }
+
   given HashBytes[TimeSpan] with {
     def hashBytes(a: TimeSpan): Array[Byte] =
       HashBytes[Long].hashBytes(a.toMicroseconds)
   }
 
-}
\ No newline at end of file
+  given [A](using HashBytes[A]): HashBytes[Option[A]] with {
```

Comment:
These behavior for the hash of Option made sense in GuideService. If it doesn't make sense globally, it could be localized there.

Reply:
Seems good to me.